examples/2d/elixir_ape_unstructured_quad_gauss_wall.jl

```diff
@@ -0,0 +1,66 @@
+
+using Downloads: download
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the acoustic perturbation equations
+
+equations = AcousticPerturbationEquations2D(v_mean_global=(0.0, -0.5), c_mean_global=1.0,
+                                            rho_mean_global=1.0)
+
+# Create DG solver with polynomial degree = 4 and (local) Lax-Friedrichs/Rusanov flux
+solver = DGSEM(polydeg=4, surface_flux=flux_lax_friedrichs)
+
+# Create unstructured quadrilateral mesh from a file
+default_mesh_file = joinpath(@__DIR__, "mesh_five_circles_in_circle.mesh")
+isfile(default_mesh_file) || download("https://gist.githubusercontent.com/andrewwinters5000/3c79baad6b4d73bb26ec6420b5d16f45/raw/22aefc4ec2107cf0bffc40e81dfbc52240c625b1/mesh_five_circles_in_circle.mesh",
+                                       default_mesh_file)
+mesh_file = default_mesh_file
+
+mesh = UnstructuredQuadMesh(mesh_file)
+
+initial_condition = initial_condition_gauss_wall
+
+#boundary_condition_gauss = BoundaryConditionDirichlet(initial_condition)
```

Comment:
```suggestion
```
Or is there a reason to keep this?

Reply:
whoops, I thought I caught all of the floatsom from when I was experimenting with the tests. I will remove it